### DISCUSSION TITLE
Issue with Ubuntu 18

### DISCUSSION THREAD

**Discussion by Paul-rs1** [apertium/organisation#37] [Q&A] [answered]
Hi,
I followed the installation guide for Apertium core, but I encountered errors when running sudo apt-get -f install apertium-all-dev. The error message stated: "Failed to fetch ... bionic1_amd64..." I checked the following URL: "http://apertium.projectjj.com/apt/release/pool/main/", but I couldn't find any bionic packages.

Is this ubuntu version not longer supported and is there a workaround for this issue?

Steps to reproduce:

1. `curl -sS https://apertium.projectjj.com/apt/install-release.sh | sudo bash` # same issue for nightly version
2. sudo apt-get -f install apertium-all-dev

System: Ubuntu 18.04

**Comment by TinoDidriksen** [comment, marked as answer]
18.04 went EOL at the end of May: https://ubuntu.com/blog/18-04-end-of-standard-support

Upgrade to a supported version of Ubuntu, ideally latest LTS.

**Reply by Paul-rs1** [reply]
Thank you for the prompt response! 👍
